src/main/kotlin/io/kni/thingoo/backend/entities/EntityServiceImpl.kt

```diff
@@ -20,4 +22,16 @@ class EntityServiceImpl(
                 ApiErrorCode.ENTITIES_003.throwException()
             }
     }
+
+    override fun updateEntity(updateEntityDto: UpdateEntityDto, id: Int): EntityDto {
```

Comment:
It probably depends on the convention and is a really little detail but in my opinion, when you give id as the first param and then updateEntityDto it seems more natural for me.

Reply:
And just for that single reason I've assigned two reviewers here :laughing: I'll change to id-first approach both here and https://github.com/ThingooKNI/backend/pull/68
Thanks

Reply:
Resolved in https://github.com/ThingooKNI/backend/pull/69/commits/c106176f4680f993798818c369a69eb3b062e26c